### ISSUE TITLE
NVDA does not read  UTorrent 2.2.1  trackers list

### ISSUE THREAD

**Issue by enessaribas** [nvaccess/nvda#10533]
### Steps to reproduce:
1. with latest NVDA master, focus an active torrent in UTorrent, and shift tab to the tabs, and switch to the trackers with the left and right arrow keys. 
2. Tab to the trackers list.
### Actual behavior:
NVDA says list item when arrowing through the trackers.
### Expected behavior:
NVDA reads name of the tracker, such as DHT.
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
Tested with latest master
#### Windows version:
windows 10, version 1909
#### Name and version of other software in use when reproducing the issue:
UTorrent 2.2.1
#### Other information about your system:
Utorrent 2.2.1 is installed as well.
#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
this is not the case with 2019.2, pre-py3 builds of NVDA. This is a regression.

**Comment by Brian1Gaff** [comment]
If you can find some other instances where this sort of thing happens, it 
might help people prioritise it higher up the pecking order, perhaps.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
-----

**Comment by lukaszgo1** [comment]
cc @leonardder  @michaelDCurran

**Comment by leonardder** [comment]
I think it would help to know where the regression started to occur.

**Comment by lukaszgo1** [comment]
I cannot reproduce this with latest Alpha on Windows 7. @enessaribas Is there any particular reason why you are using so old version of UTorrent?

**Comment by enessaribas** [comment]
hello,
The reason for this is accessibility. It isn't practical for me to test all possible alphas of NVDA to determine the regression. Is there a way for me to provide more info about this? This definately works on pre-py3 alphas of NVDA.

**Comment by lukaszgo1** [comment]
Could you please disable all add-ons, change log level to debug, try to reproduce the issue and upload a full log?

**Comment by enessaribas** [comment]
hello,
Here it is.
[nvda.log](https://github.com/nvaccess/nvda/files/3889365/nvda.log)

**Comment by lukaszgo1** [comment]
Unfortunately there is nothing useful in your log. Could you please focus one of the list items, press NVDA+F1, and paste developer info for it here.

**Comment by enessaribas** [comment]
Developer info for navigator object:
name: ''
role: ROLE_LISTITEM
roleText: None
states: STATE_INVISIBLE, STATE_OFFSCREEN, STATE_FOCUSED, STATE_FOCUSABLE, STATE_SELECTED, STATE_SELECTABLE
isFocusable: True
hasFocus: True
Python object: <NVDAObjects.Dynamic_DuplicateFocusListViewListItemIAccessible object at 0x014FB890>
Python class mro: (<class 'NVDAObjects.Dynamic_DuplicateFocusListViewListItemIAccessible'>, <class 'appModules.utorrent.DuplicateFocusListView'>, <class 'NVDAObjects.IAccessible.sysListView32.ListItem'>, <class 'NVDAObjects.behaviors.RowWithFakeNavigation'>, <class 'NVDAObjects.behaviors.RowWithoutCellObjects'>, <class 'NVDAObjects.IAccessible.sysListView32.ListItemWithoutColumnSupport'>, <class 'NVDAObjects.IAccessible.IAccessible'>, <class 'NVDAObjects.window.Window'>, <class 'NVDAObjects.NVDAObject'>, <class 'documentBase.TextContainerObject'>, <class 'baseObject.ScriptableObject'>, <class 'baseObject.AutoPropertyObject'>, <class 'object'>)
description: None
location: RectLTWH(left=231, top=498, width=0, height=19)
value: None
appModule: <'utorrent' (appName 'utorrent', process ID 6756) at address 55613b0>
appModule.productName: 'µTorrent'
appModule.productVersion: '2.2.1.25110'
TextInfo: <class 'NVDAObjects.NVDAObjectTextInfo'>
windowHandle: 1180638
windowClassName: 'SysListView32'
windowControlID: 1087
windowStyle: 1342246921
extendedWindowStyle: 516
windowThreadID: 2404
windowText: ''
displayText: ''
IAccessibleObject: <POINTER(IAccessible) ptr=0x5a9c990 at 7307990>
IAccessibleChildID: 2
IAccessible event parameters: windowHandle=1180638, objectID=-4, childID=2
IAccessible accName: '[Local Peer Discovery]'
IAccessible accRole: ROLE_SYSTEM_LISTITEM
IAccessible accState: STATE_SYSTEM_SELECTED, STATE_SYSTEM_FOCUSED, STATE_SYSTEM_INVISIBLE, STATE_SYSTEM_OFFSCREEN, STATE_SYSTEM_FOCUSABLE, STATE_SYSTEM_SELECTABLE, STATE_SYSTEM_MULTISELECTABLE, STATE_SYSTEM_VALID (20021254)
IAccessible accDescription: 'Status: not allowed, Seeds: 0, Peers: 0, Downloaded: 0'
IAccessible accValue: None

**Comment by leonardder** [comment]
This is likely a regression caused by #9873. However, given that @lukaszgo1 can't reproduce this on Alpha with a newer version of UTorrent, I'm hesitant cto change things here.

@enessaribas: Getting developer info for the list item turned out to be pretty helpful. Could you do the same, but with NVDA 2019.2.1?

**Comment by lukaszgo1** [comment]
The problem is that I was not able to reproduce this not only with the latest version of Utorrent, but also with the version 2.2.1 which is used by @enessaribas. This suggests that there is another variable at play here.

**Comment by leonardder** [comment]
I think theoretically, it could be that the column widths in the list of @enessaribas' client are changed to 0. So, an interesting question would be, is there something visible visually, or is the list also collapsed visually? In the latter case, #9873 is definitely proving to be a future rather than a regression 😁

**Comment by enessaribas** [comment]
Hello,
I apologize for the slow response.  Here is the developer info with 2019.2.1.  I would also like to point out that visually these columns are visible.

Developer info for navigator object:
name: u'[Peer Exchange]; not allowed; 0; 0; 0'
role: ROLE_LISTITEM
states: STATE_INVISIBLE, STATE_OFFSCREEN, STATE_FOCUSED, STATE_SELECTED, STATE_FOCUSABLE, STATE_SELECTABLE
isFocusable: True
hasFocus: True
Python object: <NVDAObjects.Dynamic_DuplicateFocusListViewListItemIAccessible object at 0x0564ED30>
Python class mro: (<class 'NVDAObjects.Dynamic_DuplicateFocusListViewListItemIAccessible'>, <class 'appModules.utorrent.DuplicateFocusListView'>, <class 'NVDAObjects.IAccessible.sysListView32.ListItem'>, <class 'NVDAObjects.behaviors.RowWithFakeNavigation'>, <class 'NVDAObjects.behaviors.RowWithoutCellObjects'>, <class 'NVDAObjects.IAccessible.sysListView32.ListItemWithoutColumnSupport'>, <class 'NVDAObjects.IAccessible.IAccessible'>, <class 'NVDAObjects.window.Window'>, <class 'NVDAObjects.NVDAObject'>, <class 'documentBase.TextContainerObject'>, <class 'baseObject.ScriptableObject'>, <class 'baseObject.AutoPropertyObject'>, <type 'object'>)
description: None
location: RectLTWH(left=231, top=517, width=0, height=19)
value: None
appModule: <'utorrent' (appName u'utorrent', process ID 1880) at address 54b2ed0>
appModule.productName: u'\xb5Torrent'
appModule.productVersion: u'2.2.1.25110'
TextInfo: <class 'NVDAObjects.NVDAObjectTextInfo'>
windowHandle: 459344
windowClassName: u'SysListView32'
windowControlID: 1087
windowStyle: 1342246921
extendedWindowStyle: 516
windowThreadID: 4104
windowText: u''
displayText: u''
IAccessibleObject: <POINTER(IAccessible) ptr=0x61a0868 at 5304ad0>
IAccessibleChildID: 3
IAccessible event parameters: windowHandle=459344, objectID=-4, childID=3
IAccessible accName: u'[Peer Exchange]'
IAccessible accRole: ROLE_SYSTEM_LISTITEM
IAccessible accState: STATE_SYSTEM_SELECTABLE, STATE_SYSTEM_OFFSCREEN, STATE_SYSTEM_SELECTED, STATE_SYSTEM_MULTISELECTABLE, STATE_SYSTEM_INVISIBLE, STATE_SYSTEM_FOCUSED, STATE_SYSTEM_FOCUSABLE, STATE_SYSTEM_VALID (20021254)
IAccessible accDescription: u'Status: not allowed, Seeds: 0, Peers: 0, Downloaded: 0'
IAccessible accValue: None

**Comment by leonardder** [comment]
This already has STATE)INVISIBLE and STATE)OFFSCREEN, so there's 
definitely something wrong in UTorrent.

Have you tried removing your user configuration?

**Comment by enessaribas** [comment]
Which user configuration?? Additionally, sighted people can see the client properly and I have no issues with Utorrent other than this regression. If this change will not be reverted, could you add an option to disable it for certain applications?

**Comment by lukaszgo1** [comment]
@enessaribas I believe @leonardder ment your user configuration of Utorrent from %apdata%. Could you cut it from there to some safe location, and try to reproduce this?

**Comment by enessaribas** [comment]
Hello,
What is the name of the file in appdata?

**Comment by lukaszgo1** [comment]
@enessaribas It is a directory called uTorrent

**Comment by enessaribas** [comment]
Hello,
I am aware of this. However, I cannot find the specific file. The directory contains many .torrent files that were created or downloaded by the client. There is no file called userconfig, or config.

**Comment by lukaszgo1** [comment]
Simply cut the entire directory somewhere else.

**Comment by enessaribas** [comment]
This appears to be an issue on my system. It does not occur when config is removed, which confirms this. I see I have to eventually clear out the config
